report/include/scp/report.h

```diff
@@ -55,6 +74,28 @@ enum class log {
     TRACEALL,
     DBGTRACE = TRACEALL
 };
+
+static const std::array<sc_core::sc_severity, 8> severity = {
+    sc_core::SC_FATAL,   // scp::log::NONE
+    sc_core::SC_FATAL,   // scp::log::FATAL
+    sc_core::SC_ERROR,   // scp::log::ERROR
+    sc_core::SC_WARNING, // scp::log::WARNING
+    sc_core::SC_INFO,    // scp::log::INFO
+    sc_core::SC_INFO,    // scp::log::DEBUG
+    sc_core::SC_INFO,    // scp::log::TRACE
+    sc_core::SC_INFO     // scp::log::TRACEALL
+};
+static const std::array<sc_core::sc_verbosity, 8> verbosity = {
```

Comment:
Same here as with `severity`

Reply:
I think while I was playing around, I moved these un-necissarily I'll put them back to where they were.... (in .cpp) No need for them to be exposed in the .h